variables.tf

```diff
@@ -0,0 +1,25 @@
+variable "prefix" {
+  description = "The prefix used for all resources in this environment"
+}
+
+variable "github_client_id" {
+  description = "The client id for github OAuth"
+  sensitive = true
+}
+
+variable "github_client_secret" {
+  description = "The client secret for github OAuth"
+  sensitive = true
+}
+
+variable "flask_secret_key" {
+  sensitive = true
+}
+
+variable "flask_env" {
+  description = "The environment for the flask app"
```

Comment:
This feels like the sort of argument that might benefit from a default value since it's almost always going to be the same